### PR TITLE
coreos: Copy project, team structure from metering/, add cosa buildroot 

### PR DESCRIPTION
**Pull request by cgwalters**
This is a continuation of https://github.com/openshift/release/pull/4979 but better because we're copying the better structure from the `projects/metering` subdirectory.

### CODE REVIEW

**Comment by cgwalters** [comment]
After this merges, let's do an `oc delete project/coreos` and then do a `make coreos`.

**Comment by stevekuznetsov** [comment]
Would you mind waiting on https://github.com/openshift/release/pull/5243 and using that format, so everything here is auto-applied and validated before merge?

**Comment by cgwalters** [comment]
> Would you mind waiting on #5243 and using that format, so everything here is auto-applied and validated before merge?

Sure.

**Comment by cgwalters** [comment]
Rebased :surfing_man: 
/hold cancel

File: services/coreos/rbac.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
```
Comment (stevekuznetsov) [inline services/coreos/rbac.yaml]: This will need to be an admin resource, I imagine.

Comment (cgwalters) [reply services/coreos/rbac.yaml]: I don't think so - these are [regular RoleBinding](https://kubernetes.io/docs/reference/access-authn-authz/rbac/)s scoped to the `coreos` namespace, not `ClusterRoleBindings`.

Comment (stevekuznetsov) [reply services/coreos/rbac.yaml]: OK. Our general pattern has been to consider _any_ RBAC to be admin so it's got a different workflow, but feel free to do whatever you want under `services/`

**Comment by cgwalters** [comment]
> You'll want to update the config-updater RBAC to allow operations in the coreos namespace here

Hm, that file isn't consistent; some namespaces seem to use a `config-updater` role, others don't?  It seems like it'd be cleaner if config-updater had rights to all namespaces or so?  Or maybe just namespaces which have a project file defined in this git repo?

File: services/coreos/rbac.yaml (hunk 1 above)
Comment (cgwalters) [reply services/coreos/rbac.yaml]: Tricky topic; feels like we want to cleanly separate things which are *cluster* admin vs just project RBAC?  I kept it as is, hope that's ok.

**Comment by stevekuznetsov** [comment]
> Hm, that file isn't consistent; some namespaces seem to use a config-updater role, others don't?

It isn't super important what the role is named, if that's what you mean.

> It seems like it'd be cleaner if config-updater had rights to all namespaces or so?

I don't think there's a clean way to do this and not create a super-user. 

> Or maybe just namespaces which have a project file defined in this git repo?

These change so infrequently it's been manageable to just update the rights ... WDYT @petr-muller

**Comment by petr-muller** [comment]
> These change so infrequently it's been manageable to just update the rights ...
@stevekuznetsov @bparees For now I agree it's manageable I also agree that tracking RBACs (and admin resources in general) is currently quite _"meh"_ and could be improved, especially if we start to have more of these user controlled namespaces.

**Comment by stevekuznetsov** [comment]
I think it's because it expects the namespace object to be an admin object, it does not have rights to create it

**Comment by cgwalters** [comment]
OK, passing CI now!

**Pull request merged**